public/locales/en-US/translations.json

```diff
@@ -116,19 +116,20 @@
   "transaction_EscrowCancel": "Escrow Cancel",
   "transaction_EscrowCreate": "Escrow Create",
   "transaction_EscrowFinish": "Escrow Finish",
-  "transaction_OfferCancel": "Offer Cancel",
-  "transaction_OfferCreate": "Offer Create",
   "transaction_NFTokenAcceptOffer": "NFT Accept Offer",
   "transaction_NFTokenBurn": "NFT Burn",
   "transaction_NFTokenCancelOffer": "NFT Cancel Offer",
   "transaction_NFTokenCreateOffer": "NFT Create Offer",
   "transaction_NFTokenMint": "NFT Mint",
+  "transaction_OfferCancel": "Offer Cancel",
+  "transaction_OfferCreate": "Offer Create",
```

Comment:
are you just sorting these alphabetically?

Reply:
yes - it was just slightly off and it annoyed me

Reply:
(I also added a `TicketCreate` translation in this file)